apps/arena/lib/arena/game_updater.ex

```diff
@@ -458,6 +491,18 @@ defmodule Arena.GameUpdater do
     {:noreply, state}
   end
 
+  def handle_info(:pick_default_bouty_for_missing_players, state) do
+    Enum.each(state.game_state.players, fn {player_id, player} ->
+      if not player.aditional_info.picked_bounty and not Enum.empty?(player.aditional_info.bounties) do
```

Comment:
These bounties are the ones that we randomized from the bounty_updater list for that player, if for some reason this list is empty e.g. we don't have any bounty available, this operation will fail since we cannot get a random value from an empty list

Reply:
Nice. Thanks for the explanation.